app/views/likes/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: like, class: "d-inline-block") do |form| %>
+<%= form_with(model: like, class: "d-inline-block", data: {remote: true}) do |form| %>
```

Comment:
I think, you can replace `data: {remote: true}` with just `remote: true`.

Reply:
^^